tests/unit/plugins/relay/api/test-relay-api-msg.cpp

```diff
@@ -578,3 +578,20 @@ TEST(RelayApiMsg, HotlistToJson)
 
     gui_hotlist_remove_buffer (gui_buffers, 1);
 }
+
+/*
+ * Tests functions:
+ *   relay_api_msg_completion_to_json
+ */
+
+TEST(RelayApiMsg, CompletionToJson)
+{
+    cJSON *json; //, *json_obj, *json_count;
```

Comment:
It seems this test is not complete, and some commented variables are still there.
Please make a complete test by building a completion JSON object with a real completion.

You should be able to simulate a completion from user on core buffer with such code:

```C
gui_buffer_set (gui_buffers, "input", "/co"); 
gui_buffer_set (gui_buffers, "input_pos", "3");
gui_input_complete_next (gui_buffers);
```